ocs_ci/ocs/constants.py

```diff
@@ -1544,6 +1544,7 @@
 # the pattern "/registry/" match the test path and so the test belongs to
 # Magenta squad.
 SQUADS = {
+    "Aqua": ["/lvmo/"],
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/master/pytest.ini#L45
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L401

We can add here as well for marker usage